src/constants.ts

```diff
@@ -0,0 +1,26 @@
+
+/*
+ * Copyright 2024-2025 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import * as path from 'path';
+
+export const EXTENSION_ROOT_DIR = path.resolve(path.dirname(__dirname));
+export const WRAPPERS_DIR = path.join(EXTENSION_ROOT_DIR, 'wrappers');
+export const CHPL_WRAPPER = path.join(WRAPPERS_DIR, "chplWrapper.sh");
```

Comment:
Nit: mixed use of double and single quotes, perhaps we could use an opinionated linter to help with this?

Reply:
There is an opinionated linter and I ran it and fixed all the issues :smile: